charts/app-config/values-production.yaml

```
@@ -917,6 +917,38 @@ govukApplications:
             name: email-alert-auth
             key: token
 
+- name: email-alert-monitoring
+  helmValues:
+    appEnabled: false
+    uploadAssets:
+      enabled: false
+    rails:
+      enabled: false
+    cronTasks:
+      - name: run_medical_alerts
+        task: "run_medical_alerts"
+        schedule: "21 51 * * *"
```

This cron is invalid, think its meant to be:
```suggestion
        schedule: "21,51 * * * *"
```

---

Good spot @theseanything